crates/optimism/payload/Cargo.toml

```diff
@@ -27,13 +27,14 @@ reth-payload-util.workspace = true
 reth-payload-primitives = { workspace = true, features = ["op"] }
 reth-basic-payload-builder.workspace = true
 reth-chain-state.workspace = true
+reth-payload-validator.workspace = true
 reth-trie.workspace = true
 
 # op-reth
 reth-optimism-consensus.workspace = true
 reth-optimism-evm.workspace = true
 reth-optimism-forks.workspace = true
-reth-optimism-primitives.workspace = true
+reth-optimism-primitives = { workspace = true, features = ["reth-codec"] }
```

Comment:
this is an annoying side-effect of the storage-api crate.

working a fix for this, but for now this is okay